packages/core/src/transport/HttpOutboundTransport.ts

```diff
@@ -76,6 +77,11 @@ export class HttpOutboundTransport implements OutboundTransport {
         this.logger.debug(`Response received`, { responseMessage, status: response.status })
 
         try {
+          if (!isValidJweStucture(responseMessage)) {
+            throw new Error(
+              `Received a response from the other agent but the structure of the incoming message is not a DIDComm message: ${responseMessage}`
+            )
+          }
```

Comment:
After thinking about it for a bit, maybe we should log n error here and return? It's not fatal and we can just move on.

Reply:
Also, `responseMessage` is of type string here, not json.